app/main.go

```diff
@@ -73,7 +74,17 @@ func sync(w http.ResponseWriter, r *http.Request) {
 	}
 	defer stats.Close()
 
-	if errs := tsbridge.UpdateAllMetrics(ctx, config, sd, stats); errs != nil {
+	p, err := strconv.Atoi(os.Getenv("UPDATE_PARALLELISM"))
```

Comment:
https://play.golang.org/p/rb9vttBTHoD

This will cause problems when UPDATE_PARALLELISM is unset. I know you set it by default to 1 in the app YAML, but you probably want to also default to 1 in the code when the env var is unset…

Reply:
I'd rather not complicate the code by hard-coding default values. If `UPDATE_PARALLELISM` is unset, the function will return an error, which seems ok?